src/main/java/com/synopsys/integration/jenkins/ChangeSetFilter.java

```diff
@@ -85,7 +85,7 @@ private boolean shouldInclude(final String filePath) {
         // ChangeLogSet.AffectedFile getPath is normalized to use the / separator
         final String fileName;
         if (filePath.contains("/")) {
-            fileName = filePath.substring(filePath.lastIndexOf("/"));
+            fileName = filePath.substring(filePath.lastIndexOf("/") + 1);
         } else {
             fileName = filePath;
         }
```

Comment:
This can be replaced by
``` java
String fileName;
        if (filePath.contains("/")) {
            fileName = StringUtils.substringAfter(filePath, "/");
        } else {
            fileName = filePath;
        }
```